P/PowerModels/Compat.toml

```diff
@@ -10,18 +10,18 @@ JSON = "0"
 ["0.2.0"]
 MathProgBase = "0"
 
-["0.2.1-0"]
-MathProgBase = "0.5.4-0"
-
 ["0.2.1-0.2"]
 JuMP = "0.14"
 
-["0.2.2-0"]
-JSON = "0.6-0"
+["0.2.1-0.9.5"]
+MathProgBase = "0.5.4-0"
```

Comment:
Please keep the requirement on MathProgBase

Reply:
Why?  Usage of MPB in the core package was removed in this release.